docs/writing-docs/docs-page.md

```diff
@@ -54,30 +54,33 @@ If you want to organize your documentation differently for component groups, we
 Replace the DocsPage template with your own to customize its contents.
 
 ### With null to remove docs
- 
+
 Override the `docs.page` [parameter](../writing-stories/parameters.md) with `null` to remove its contents.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
-  paths={[
-    'common/button-story-disable-docspage-component.js.mdx',
-  ]}
+paths={[
+'common/button-story-disable-docspage-component.js.mdx',
+]}
 />
 
 <!-- prettier-ignore-end -->
 
 ### With MDX documentation
 
-Write your documentation in MDX and update the `docs.page` [parameter](../writing-stories/parameters.md) to display it.
+Write your documentation in MDX and update the `docs.page` [parameter](../writing-stories/parameters.md) to display it. The `id` of reference follows the pattern: `category-title--name`, where:
+- `category`: is what comes **before** the `/` in your `title`
+- `title`: is the story title, which comes **after** the `/` in your `title`
+- `name`: is the story variant's name (the component name, for instance)
```

Comment:
@RickCardoso if you could factor in this [docs](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#grouping) as a means of consistency I'd appreciate it.

The rationale behind this is the following. calling it a category is a bit generic/abstract. As technically what you refer to as category and title could be one and the same, it's up to the user on how he/she would group the stories in Storybook.

Also when you mention " `name`: is the story variant name...", technically that's the story itself as you can see in the snippet provided (MDX file)
.

Reply:
Done, please review